src/System.Text.Utf8String/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,27 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTrademark("")]
+
+[assembly: InternalsVisibleTo("System.Text.Utf8String.Tests,PublicKey="+
```

Comment:
Can we make the following classes public and then remove this AssemblyInfo file?
- Utf16LittleEndianCodePointEnumerable
- Utf16LittleEndianCodePointEnumerator

Other than these, what else does the Utf8String Test project need that is internal?